Lib/test/test_hashlib.py

```diff
@@ -303,6 +300,8 @@ def test_hexdigest(self):
                 self.assertIsInstance(h.digest(), bytes)
                 self.assertEqual(hexstr(h.digest()), h.hexdigest())
 
+    # TODO: RUSTPYTHON
+    @unittest.skip
```

Comment:
does this crash the interpreter or could it just be an expected failure?

Reply:
Looking at this test, I think I know how to fix it ... wait a second 🙂

Reply:
Well, it's possible but probably worth a separate PR.
The test doesn't crash the interpreter so I'll mark it as expectedFailure.